lib/findOptions.js

```diff
@@ -58,6 +58,11 @@ function readOptionsFromPackageJson (packageJsonPath) {
 
       var linter = linters[0]
       var pathToProject = path.dirname(packageJsonPath)
+      var pathToLinter = resolveFrom(pathToProject, linter)
+
+      if (!pathToLinter) {
+        return reject(new Error('linter ' + linter + ' does not seem to be installed'))
+      }
```

Comment:
I was intentionally cautious with adding errors when there was no linter active, as it is annoying to have a bunch of warnings popup when you work in a project that doesn't have a linter setup for whatever reason.

But in this case, the warning is only shown when a linter was defined, and it couldn't be found, so :+1: :)